forcetree.c

```diff
@@ -442,7 +453,7 @@ force_tree_build_single(const int firstnode, const int lastnode, const int npart
     }
 
     /* insert the pseudo particles that represent the mass distribution of other domains */
-    force_insert_pseudo_particles(lastnode);
+    force_insert_pseudo_particles(firstnode, lastnode);
```

Comment:
Feels like (firstnode, lastnode) and a few others can be bundled into a struct. TreeBuilder, e.g. but this can be addressed later.